### DISCUSSION TITLE
cargo-mutants at RustConf, September, Montreal! 🍁 🦀 🧟

### DISCUSSION THREAD

**Discussion by sourcefrog** [sourcefrog/cargo-mutants#397] [Announcements]
I'm very much much looking forward to talking about cargo-mutants at [RustConf 2024](https://rustconf.com/schedule/#wednesday) in Montreal, on September 11. 

I hope you can join me there! Come and say hi!

**Comment by dmoka** [comment]
That's impressive, congrats on this, all the best with it!
